src/Diagnostics.ModelsAndUtils/Models/GeoMaster/NetworkResourceProviderEntities.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class NetworkResourceProviderRouteTable
+    {
+        public string ProvisioningState { get; set; }
+        public bool DisableBgpRoutePropagation { get; set; }
+        public List<NetworkResourceProviderRoute> Routes { get; set; }
+        public List<NetworkResourceProviderSubnet> Subnets { get; set; }
+    }
+
+    public class NetworkResourceProviderRoute
+    {
+        public string ProvisioningState { get; set; }
+        public string NextHopIpAddress { get; set; }
+        public string AddressPrefix { get; set; }
+        public string NextHopType { get; set; }
+    }
+
+    public class NetworkAddressSpace
+    {
+        public List<string> AddressPrefixes { get; set; }
```

Comment:
Let's use `IEnumerable` for iterative needs (foreach) or `IList` if this requires `.Add()`

Reply:
That's right